bot/exts/evergreen/avatar_modification/avatar_modify.py

```diff
@@ -121,6 +121,34 @@ async def eightbit_command(self, ctx: commands.Context) -> None:
 
         await ctx.send(embed=embed, file=file)
 
+    @avatar_modify.command(name="reverse", root_aliases=("reverse",))
+    async def reverse(self, ctx: commands.Context, *, text: t.Optional[str]) -> None:
+        """Either flips your profile picture or the submitted text."""
+        if not text:
+            async with ctx.typing():
+                user = await self._fetch_user(ctx.author.id)
```

Comment:
Use may be none here if this call fails, we should deal with that similar to how the other commands do.

Reply:
use if text is None instead?

Reply:
No, I mean `self._fetch_user()` could return `None`, so you'd need to handle that case, otherwise the next line with error. The other commands in this cog already do it.

https://github.com/python-discord/sir-lancebot/blob/6dc4dcd52d7dd697a6c0ad8a7cc941eeb1d102b7/bot/exts/evergreen/avatar_modification/avatar_modify.py#L99-L102

Reply:
This hasn't been resolved.

Reply:
OH